docs/pages/blog/lab-date-pickers-to-mui-x.md

```diff
@@ -61,7 +61,7 @@ All other date and time picker components are available in `mui/x-date-pickers`
 
 ## How do I purchase a commercial license?
 
-Visit [the MUI Store](https://mui.com/store/items/material-ui-pro/) to purchase a commercial license.
+Visit [the MUI Store](/store/items/material-ui-pro/) to purchase a commercial license.
```

Comment:
For links to external docs, I would recommend an absolute URL, we have no guarantee that once this is versioned, it will keep working, e.g. https://v4.mui.com/blog/2019/